lib/cmdledit.g

```diff
@@ -751,10 +751,108 @@ GAPInfo.CommandLineEditFunctions.Functions.(INT_CHAR(' ')) :=
                 GAPInfo.CommandLineEditFunctions.Functions.SpaceDeletePrompt;
 BindKeysToGAPHandler(" ");
 
+
+BindGlobal("STANDARD_EXTENDERS", rec(
```

Comment:
This name tells me nothing about the purpose of this record. Coud you add a comment before it that briefly states what is is used for?